fw/tls.c

```diff
@@ -754,20 +761,20 @@ tfw_tls_conn_send(TfwConn *c, TfwMsg *msg)
 {
 	int r;
 	TlsCtx *tls = tfw_tls_context(c);
-	TlsIOCtx *io = &tls->io_out;
 
 	/*
 	 * Only HTTP messages go this way, other (service) TLS records are sent
 	 * by tfw_tls_send().
 	 */
-	io->msgtype = TTLS_MSG_APPLICATION_DATA;
```

Comment:
Why? I see that the TLS code does check `msgtype` against `TTLS_MSG_APPLICATION_DATA`, but now nobody sets the flag.

I do not understand what the commit fixes and and the fix seems buggy for me.